Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1

```diff
@@ -499,6 +499,7 @@ if (!($noDiscoveryService)) {
         $discoveryServiceUrl = $userEnteredDiscoveryServiceUrl
     }
 
+    $discoveryServiceUrl = $discoveryServiceUrl.TrimEnd('/')
```

Comment:
This looks good, but the following code that was added previously,  if($discoveryRegisterUrl -notmatch '/v\d+/*$') {
         $discoveryRegisterUrl = "$discoveryRegisterUrl/v1"
     }
will fail if discoveryRegisterUrl was "http://localhost/DiscoveryService/v1/Services" it will return "http://localhost/DiscoveryService/v1/Services/v1"

Reply:
where are we adding v1 if needed?

Reply:
this code is line 640 below in this script

Reply:
Passing the url as http://localhost/DiscoveryService/v1/Services, will break the fabric-install utils which should throw an error, i believe it will also break identity if it gets to the point as well since identity is also not expecting the this discovery url.  
The v1 add happens on line 638 which isn't part of this pr

Reply:
It is part of the last check in which I missed, since I was gone, and is based on the code I added for the bug. It is just an observation.